### ISSUE TITLE
Let non-aircraft have CMD_AREA_ATTACK

### ISSUE THREAD

**Issue by sprunk** [beyond-all-reason/RecoilEngine#1032]

**Comment by WatchTheFort** [comment]
Isn't this already the case? iirc artillery units have this

**Comment by sprunk** [comment]
No, it's a gadget https://github.com/beyond-all-reason/Beyond-All-Reason/blob/master/luarules/gadgets/unit_areaattack.lua

A gadget is a fine solution, but if the engine already supports it natively for bombers it may as well for everything else.

**Comment by sprunk** [comment]
After looking into the topic it turns out there's a ton of game design issues that the bomber implementation gets to avoid (see https://github.com/beyond-all-reason/RecoilEngine/pull/3275#discussion_r3852174727) so I don't think extending it is necessarily a good idea.
